en/manage-cli-advanced.md

```diff
@@ -104,6 +104,90 @@ maas $PROFILE vlan update 2 0 relay_van=5002
 ```
 
 
+## Assign a network interface to a fabric
+
+This task is easier understood when viewing CLI output with the aid of the `jq`
+utility. Go ahead and install it:
```

Comment:
Before installing it, I think it's worth quickly saying what `jq` is and why it may be helpful.